cfn-templates/cid-cfn.yml

```diff
@@ -108,29 +95,29 @@ Parameters:
   DeployCUDOSDashboard:
     Type: String
     Description: Deploy CUDOS Dashboard
-    Default: "no"
+    Default: "yes"
```

Comment:
why not to make them all no and then set respective one to yes with the url parameters from the lab link?

Reply:
can be. will do